extensions/dependencies/count_vectorizer.html

```diff
@@ -0,0 +1 @@
+<script src="{{cache_slug}}/core/templates/dev/head/domain/classifier/CountVectorizerService.js"></script>
```

Comment:
Well we can do that but these dependencies are separate and can be used in more than one interaction classifiers. For e.g. we may need SVMPredictionService whenever interaction classifier is built on SVM classifier. So it makes more sense to keep them individual as they would become part of more than one interaction classifiers.